src/julia/Matrix.jl

```diff
@@ -0,0 +1,65 @@
+###############################################################################
+#
+#   Matrix.jl : Additional AbstractAlgebra functionality for Julia Matrices
+#
+###############################################################################
+
+nrows(A::Matrix{T}) where {T} = size(A)[1]
+ncols(A::Matrix{T}) where {T} = size(A)[2]
+
+function is_zero_row(M::Matrix, i::Int)
+    for j = 1:ncols(M)
+        if !iszero(M[i, j])
+            return false
```

Comment:
There seem to be no tests for all this code. I wonder if Nemo or Hecke have any left-over test code?

Reply:
Probably. But finding this is most time a bit tedious. I can have a look in there

Reply:
Thank you!